src/hb-ot-face-table-list.hh

```diff
@@ -62,6 +62,7 @@ HB_OT_ACCELERATOR (OT, name)
 #ifndef HB_NO_STAT
 HB_OT_TABLE (OT, STAT)
 #endif
+HB_OT_TABLE (OT, meta)
```

Comment:
Remove this part until we actually start using...

I think I like to add a separate area for the rarely-used tables.  Maybe doesn't matter.  But I think currently we only expose tables that are actually used.

Reply:
I like having every table listed there, but commented out if unused.  That's the case for BASE and JSTF.  Not sure if there's other tables that need to be listed for completeness (and commented out).